test/e2e/logs_test.go

```diff
@@ -40,6 +40,15 @@ func testNodeLogs(t *testing.T) {
 						log.Printf("unable to retrieve log %s from node %s: %s", file, node.GetName(), err)
 						return false, nil
 					}
+					logInfo, err := os.Stat(file)
+					if err != nil {
+						log.Printf("unable to retrieve info for log %s from node %s: %s",
```

Comment:
This makes it seem you are running stat on the node which is not the case. `unable to get info for retrieved log %s from node %s: %s` seems more appropriate.